addons/embed_tlm_pytorch.py

```diff
@@ -126,15 +126,13 @@ def __init__(self, **kwargs):
         self.d_model = int(kwargs.get('dsz', kwargs.get('d_model', 410)))
         d_ff = int(kwargs.get('d_ff', 2100))
         d_k = kwargs.get('d_k')
-        rpr_k = kwargs.get('rpr_k')
         embed_type = kwargs.get('word_embed_type', 'positional')
-        if embed_type == 'positional':
-            x_embedding = PositionalLookupTableEmbeddings(vsz=self.vsz, dsz=self.d_model)
-        elif embed_type == 'learned-positional':
-            x_embedding = LearnedPositionalLookupTableEmbeddings(vsz=self.vsz, dsz=self.d_model)
+        rpr_k = kwargs.get('rpr_k')
+        x_embedding = create_embeddings(vsz=self.vsz, dsz=self.d_model, embed_type=embed_type)
```

Comment:
why is this change required?

Reply:
when we use `rpr_k`, there's no need to use positional embeddings any more. This change enables using `default` + `rpr_k`.